extensions/notebook/src/book/remoteBook.ts

```diff
@@ -14,7 +14,11 @@ export abstract class RemoteBook {
 
 	constructor(public remotePath: URL, protected _asset?: IAsset) {
 		this.remotePath = remotePath;
-		this.outputChannel = vscode.window.createOutputChannel(loc.msgTaskName);
+		this.outputChannel = this.createOutputChannel();
+	}
+
+	public createOutputChannel(): vscode.OutputChannel {
+		return this.outputChannel || (this.outputChannel = vscode.window.createOutputChannel(loc.msgTaskName));
```

Comment:
This isn't a good name for an output channel. And in general we should try to only have one output channel per extension (if any). 

Notebooks already has an output channel so you should use that one instead.

Reply:
It seems that the notebooks output channel is created in here in the [JupyterController ](https://github.com/microsoft/azuredatastudio/blob/main/extensions/notebook/src/jupyter/jupyterController.ts#L49). Do you think I should register the command here, instead of bookTreeView?

Reply:
What command?

Reply:
In [here ](https://github.com/microsoft/azuredatastudio/blob/eccf3cf5fe8f60d04d70096bc94c654cab60d81a/extensions/notebook/src/jupyter/jupyterController.ts#L88) the outputChannel is passed as JupyterInstallation, so maybe adding the command openRemoteBook and pass the outputChannel? Not sure if this the right approach. Or maybe creating a new command to download GitHub resources?

Reply:
Currently, the openRemoteBook logic is located in BookTreeView, which doesn't have an output channel